app/assets/stylesheets/application.css

```diff
@@ -28,12 +29,12 @@ img{
 }
 
 body{
-	background: #FFCABC;
+	background: #FFE8DD;
 }
 
 .banner {
     position: relative;
-    box-shadow: grey 5px 5px 20px;
+    box-shadow: #313233 5px 5px 20px;
```

Comment:
4 space tab, or 2 space tab? Decide and make sure to [be consistent](https://google.github.io/styleguide/htmlcssguide.html#Parting_Words) with surrounding code.